src/CoreWCF.NetTcp/tests/ClientContract/ITestService.cs

```diff
@@ -43,5 +43,13 @@ public interface ITestService
         [OperationContract(Name = "EchoForImpersonation", Action = Constants.OPERATION_BASE + "EchoForImpersonation",
         ReplyAction = Constants.OPERATION_BASE + "EchoForImpersonationResponse")]
         string EchoForImpersonation(string echo);
+
+        [OperationContract(Name = "EchoForAuthorizarionOneRole", Action = Constants.OPERATION_BASE + "EchoForAuthorizarionOneRole",
+        ReplyAction = Constants.OPERATION_BASE + "EchoForAuthorizarionOneRoleResponse")]
```

Comment:
Typo, Autoriazarion -> Authorization

Reply:
can't find

Reply:
Sorry, I should have copy and pasted it instead of trying to retype it. The incorrect text says Authorizarion

OperationContract(Name = "EchoForAuthori**zarion**OneRole